x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_logic.ts

```diff
@@ -73,7 +74,7 @@ interface ResultSettingsValues {
   nonTextResultFields: FieldResultSettingObject;
   serverResultFields: ServerFieldResultSettingObject;
   resultFieldsAtDefaultSettings: boolean;
-  resultFieldsEmpty: boolean;
+  isEverythingDisabled: boolean;
```

Comment:
I kind of liked `resultFieldsEmpty` as a var name haha 🤔 `isEverythingDisabled` makes me think the inputs were deliberately disabled rather than cleared/unchecked/empty. Seriously not a huge deal tho, I'm being nitpicky lol